src/main/java/iudx/resource/server/database/elastic/ConvertElasticResponseToCsv.java

```diff
@@ -0,0 +1,123 @@
+package iudx.resource.server.database.elastic;
+
+import co.elastic.clients.elasticsearch.core.search.Hit;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class ConvertElasticResponseToCsv extends AbstractConvertElasticSearchResponse {
+  //    private static final Logger LOGGER =
+  // LogManager.getLogger(ConvertElasticResponseToCSV.class);
+  private File csvFile;
+  private FileWriter fileWriter;
+
+  /**
+   * Converts JSON records from Elasticsearch batch response to CSV format and writes it into a CSV
+   * file
+   *
+   * @param file File to write csv records
+   */
+  public ConvertElasticResponseToCsv(File file) {
+    super(file);
+    this.csvFile = file;
```

Comment:
No need for separate duplicate file variable, use the existing `file` variable

Reply:
Changed it here : [Refactor files](https://github.com/datakaveri/iudx-resource-server/pull/425/commits/3f2f16ce5bedb80fe7235b2d794c3393641dcc2c)